test/extended/jenkins/plugin.go

```diff
@@ -203,9 +203,11 @@ var _ = g.Describe("[jenkins][Slow] openshift pipeline plugin", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By("get build console logs and see if succeeded")
-			err = waitForJenkinsActivity(fmt.Sprintf("http://%s/job/test-plugin-job/1/console", hostPort), password, "Finished: SUCCESS", 200)
+			err = waitForJenkinsActivity(fmt.Sprintf("http://%s/job/test-plugin-job/1/console", hostPort), "GET", password, "Finished: SUCCESS", 200)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
+			g.By("trigger test connection logic, check for success")
+			immediateInteractionWithJenkins(fmt.Sprintf("http://1.3.4.5/job/test-plugin-job/descriptorByName/com.openshift.jenkins.plugins.pipeline.OpenShiftScaler/testConnection", "POST", hostPort), "POST", password, fmt.Fprint(body, "apiURL=&authToken="), 200)
```

Comment:
ok cool ... just need to convert this one to the `waitFor..` call where I'm assuming you'll be able to search for "Connection successful" via https://github.com/openshift/jenkins-plugin/blob/master/src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftPluginDescriptor.java#L131 once v1.0.24 of the plugin is available and we've updated the centos image.